frontend/pages/ManageControlsPage/ManageControlsPage.tsx

```diff
@@ -109,10 +98,7 @@ const ManageControlsPage = ({
   const onConnectClick = () => router.push(PATHS.ADMIN_INTEGRATIONS_MDM);
 
   const renderBody = () => {
-    if (isLoadingMdmApple) {
-      return <Spinner />;
-    }
-    return mdmApple ? (
+    return isPremiumTier && config?.mdm.enabled_and_configured ? (
```

Comment:
I believe this shouldn't be restricted to isPremiumTier –  Fleet free should still display the tabbed navigation under controls, then each tab has some kind of upsell message (for example, [for macOSUpdates](https://www.figma.com/file/hdALBDsrti77QuDNSzLdkx/%F0%9F%9A%A7-Fleet-EE-(dev-ready%2C-scratchpad)?node-id=12195%3A329751&t=0NUCnKuhZpiimJja-1)).

There was some confusion about this initially, and [@noahtalerman confirmed](https://github.com/fleetdm/fleet/issues/9349#issuecomment-1397101290) that the empty states should be tab-level, not page level, if mdm is enabled on Free.

Reply:
gotcha, just double checking: this will change the existing behavior of the page, on `main` we're only doing the API call if the tier is premium:

https://github.com/fleetdm/fleet/blob/074ea7dc8dcac1b7e2a9b0120d7157434407b86d/frontend/pages/ManageControlsPage/ManageControlsPage.tsx#L64-L71


is removing the conditional good enough?

Reply:
Yes I think so. I guess we never caught that because we haven't yet tested the Free && MDM Enabled state, thanks to our new best friend: https://github.com/fleetdm/fleet/issues/10299 😅

Reply:
I think we still need to implement those tab-level upsell states in macOS update and macOS settings tabs for Free users. cc @ghernandez345

Reply:
gotcha, thanks! I removed the conditional.